setup.py

```diff
@@ -24,15 +24,15 @@ def run(self):
             sys.exit(-1)
 
 
-if sys.version_info.major == 3 and sys.version_info.minor == 3:
+if sys.version_info.major == 3 and sys.version_info.minor == 5:
```

Comment:
We only want to check for major version because we’re differentiating between Python 2.x and Python 3.x. In case of eg 3.6, the check as is would take the Py2 path, which would be incorrect.